src/main/todos.ts

```diff
@@ -1,22 +1,38 @@
-import { getBasecampProjectUrl, sendBasecampPostRequest, sendBasecampPutRequest } from "./basecamp";
+import { getBasecampProjectUrl, PROJECT_ID, sendBasecampPostRequest, sendBasecampPutRequest } from "./basecamp";
+import { BasecampRequestMissingError } from "./error/basecampRequestMissingError";
+import { getRoles, RoleStatus } from "./role";
 
 const TODOLISTS_PATH: string = '/todolists/';
 const TODO_PATH: string = '/todos/';
+const RECORDINGS_PATH: string = '/recordings/';
 const JSON_PATH: string = '.json';
 const TODO_JSON_PATH: string = '/todos' + JSON_PATH;
+const TRASHED_TODO_JSON_PATH: string = '/status/trashed' + JSON_PATH;
+
 export const TODOLIST_ID: string= "7865336721";
 
+const DEFAULT_TODOLIST_IDENTIFIER: TodolistIdentifier = {
+    projectId: PROJECT_ID,
+    todolistId: TODOLIST_ID
+};
+
```

Comment:
It looks like this is a duplicate of the definition that is being exported from `main.ts`